alf/drivers/policy_driver.py

```diff
@@ -125,17 +125,17 @@ def _training_summary(self, training_info, loss_info, grads_and_vars):
                                                   self._train_step_counter)
         if self._debug_summaries:
             common.add_action_summaries(training_info.action,
-                                        self.env.action_spec())
+                                        self._env.action_spec())
```

Comment:
why make this change?

Reply:
You are right, no need for this.  I misread the code.

Reply:
This file is a bit confusing, sometimes it uses .env sometimes ._env.  I was trying to make it consistent.